### ISSUE TITLE
Using a -2 drive

### ISSUE THREAD

**Issue by MinerGuy62** [madMAx43v3r/chia-plotter#641]
Has anyone noticed improvement using -2?

And what is the best option for tmp2?  I have 2 1TB NVME and slow 2TB NVME and I was considering using the 2TB as the tmp2 so the 1TB's can and take on the writes or the bulk of them

**Comment by gryan315** [comment]
Yes. The -2 drive should be the faster drive because it does brunt of the work in making the tables. RAM tmpfs seems to be the best -2 for speed, but I've also used a 16x 10k SAS drive raid array as -2 and NVME as -t and got 2 minute slower results than the ramdisk.

**Comment by shilailin** [comment]
> Yes. The -2 drive should be the faster drive because it does brunt of the work in making the tables. RAM tmpfs seems to be the best -2 for speed, but I've also used a 16x 10k SAS drive raid array as -2 and NVME as -t and got 2 minute slower results than the ramdisk.

how many minutes of one plot when you using ramdisk ?

**Comment by gryan315** [comment]
With -t as NVME and -2 as tmpfs: 26 minutes. With -t as NVME and -2 as 16 drive array: 28 minutes. 2x E5 2678v3, 24 cores total.

**Comment by lBoneCrusherl** [comment]
> With -t as NVME and -2 as tmpfs: 26 minutes. With -t as NVME and -2 as 16 drive array: 28 minutes. 2x E5 2678v3, 24 cores total.

With -t as NVME and -2 as RamDisk: 38 minutes with x2 intel xeon 4210 Silver Total 20 cores 40 threads. ıs that normal? I monitored that when phase 1 my cpus reach up just 2,7Ghz but their turbo mod 3,2Ghz.

**Comment by gryan315** [comment]
What operating system, what model NVME, what -r setting, and how many buckets? I would expect faster since those newer CPUs are right around mine in terms of specs and performance.

**Comment by lBoneCrusherl** [comment]
> What operating system, what model NVME, what -r setting, and how many buckets? I would expect faster since those newer CPUs are right around mine in terms of specs and performance.

-n 10 -r 40 -v 128 -u 256 -t S:\Madmax\ -2 Z:\Temp\ -d Q:\Madmax\ -p  -f 
S: Samsung Pro 970 1TB
-2: Ramdisk
-d: generally 4tb hdd
Operating System: Windows 10
